src/plugins/intel_cpu/src/nodes/conv.cpp

```diff
@@ -1650,26 +1639,13 @@ void Convolution::appendZeroPointsArgs() {
 
 // Due to performance issue, brgconv will only be enabled by default:
 // 1, static shape(dynamic shape may change weights layout if the input shape changes and cause performance issue: 86948)
-// 2, support amx except having input zero point.
-// 3, support avx512 without legacy postops/per channel zero point when avx512
+// 2, hw supports avx512+
```

Comment:
`2,` should be removed?

Reply:
still have the limitation yet, after avx2 enabled it will change to avx2+.